Remotion/Web/Development.WebTesting.IntegrationTests/BrowserLog/BrowserLogUtilityTest.cs

```diff
@@ -0,0 +1,161 @@
+// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+// 
+// The re-motion Core Framework is free software; you can redistribute it 
+// and/or modify it under the terms of the GNU Lesser General Public License 
+// as published by the Free Software Foundation; either version 2.1 of the 
+// License, or (at your option) any later version.
+// 
+// re-motion is distributed in the hope that it will be useful, 
+// but WITHOUT ANY WARRANTY; without even the implied warranty of 
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+// GNU Lesser General Public License for more details.
+// 
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+using System;
+using System.Linq;
+using Moq;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using OpenQA.Selenium;
+using Remotion.Web.Development.WebTesting.BrowserSession;
+using Remotion.Web.Development.WebTesting.Configuration;
+using Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure.BrowserLog;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests.BrowserLog;
+
+[TestFixture]
+public class BrowserLogUtilityTest
+{
+  private Mock<IWebTestSettings> _webTestSettings;
+  private IWebTestSettings _regularSettings;
+
+  [SetUp]
+  public void Setup ()
+  {
+    _regularSettings = WebTestSettings.Current;
+    var loggerFactory = new Mock<Microsoft.Extensions.Logging.ILoggerFactory>();
+    var logger = new Mock<Microsoft.Extensions.Logging.ILogger>();
+    loggerFactory.Setup(_ => _.CreateLogger(It.IsAny<string>())).Returns(logger.Object);
+
+    _webTestSettings = new Mock<IWebTestSettings>();
+    _webTestSettings.Setup(_ => _.LoggerFactory).Returns(loggerFactory.Object);
+
+    WebTestSettings.SetCurrent(_webTestSettings.Object);
+
+  }
+
+  [TearDown]
+  public void TearDown ()
+  {
+    WebTestSettings.SetCurrent(_regularSettings);
+  }
+
+  [Test]
+  [TestCase(LogLevel.Off, false)]
+  [TestCase(LogLevel.Severe, true)]
+  [TestCase(LogLevel.Warning, true)]
+  [TestCase(LogLevel.Info, true)]
+  [TestCase(LogLevel.Debug, true)]
+  [TestCase(LogLevel.All, true)]
+  public void BrowserLogUtility_IsBrowserLogCheckActive_Default (LogLevel logLevel, bool expected)
```

Comment:
naming: the class name is usually not part of the test names, also for statis methods. Is there a reason I do not know?